packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -386,9 +386,7 @@ export class SummarizerNode implements IRootSummarizerNode {
 	public async refreshLatestSummary(
 		proposalHandle: string | undefined,
 		summaryRefSeq: number,
-		fetchLatestSnapshot: () => Promise<IFetchSnapshotResult>,
-		readAndParseBlob: ReadAndParseBlob,
```

Comment:
`ContainerRuntime::refreshLatestSummaryAck` still calls this with the params that are removed. You should clean up that code as well.
I would recomment removing the code in SummarizerNode in a separate PR because it will involve removing other pieces of code too. This PR can then focus on just making closing container the default.

Reply:
Also, these params should be removed from the interface.

Reply:
Yeah +1 to have a tiny PR to just switch the defaults